mirrord/intproxy/src/proxies/incoming.rs

```diff
@@ -49,6 +50,31 @@ mod port_subscription_ext;
 mod subscriptions;
 mod tasks;
 mod tcp_proxy;
+mod tls;
+
+#[derive(Default)]
+struct MirrorOrSteal<T> {
```

Comment:
Any reason to keep this so generic, since the only thing that changes is the value type of the hashmap?

Also, missing docs.